e2e/components/grid-list/grid-list.e2e.ts

```diff
@@ -1,11 +1,13 @@
 import {browser} from 'protractor';
 import {expectToExist} from '../../util/asserts';
+import {screenshot} from '../../screenshot';
 
 describe('grid-list', () => {
   beforeEach(() => browser.get('/grid-list'));
 
   it('should render a grid list container', () => {
     expectToExist('md-grid-list');
+    screenshot('');
```

Comment:
This could be improved by either having the `id` be an optional param, or making it default to `''`.